common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/microsoftsts/MicrosoftStsAuthorizationRequest.java

```diff
@@ -46,14 +46,15 @@
     private static final String TAG = MicrosoftStsAuthorizationRequest.class.getSimpleName();
 
     /* Constants */
-    private static final String CORRELATION_ID = "correlation_id";
+    private static final String CORRELATION_ID = "client-request-id";
```

Comment:
`correlationId` and `client-request-id` are not the same value

Reply:
unfortunately, correlation_id goes under client-request-id field in the header, the code is correct if that is the usage.

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-android/blob/f3861311f0ea1ee32cd30da7b548f8f701515775/msal/src/internal/java/com/microsoft/identity/client/OauthConstants.java#L79